distribution/openrct2.d.ts

```diff
@@ -3333,6 +3333,56 @@ declare global {
 
     type StaffType = "handyman" | "mechanic" | "security" | "entertainer";
 
+    /**
+     * Represents a handyman.
+     */
+    interface Handyman extends Staff {
```

Comment:
Can you add `staffType: "handyman";` here and similar changes to the other new interfaces? 

It would allow Typescript to understand this without errors or needing casting:
```ts
const peep: Staff = ...

if (peep.staffType == "handyman") {
    // Because of the 'staffType' check, Typescript now knows 'peep' is of type 'Handyman' 
    // and you can directly access its properties without any additional casts.
    peep.lawnsMown = 10;
}
```
This also makes it similar to how `BaseTileElement.type` is done in the various overrides. Note that no additional cpp changes are necessary here either. 🙂

Reply:
That is not enough, we need `type Staff = Handyman | Mechanic | Security | Entertainer;` for that. Which comes with a renaming of the original `Staff` interface to `BaseStaff`. Also, the `Entertainer` interface will be needed for that.